### ISSUE TITLE
Assembly definitions?

### ISSUE THREAD

**Issue by MuhammedResulBilkil** [justinwasilenko/Unity-Style-Guide#4]
How should we set name for Assembly Definition (.asmdef) files in unity?

**Comment by justinwasilenko** [comment]
Yeah I should add that, but my way of naming them is, after their name
space. For example:

   - ProjectName.Core.asmdef
   - ProjectName.Editor.asmdef

**Comment by MuhammedResulBilkil** [comment]
How about Addressables?

**Comment by justinwasilenko** [comment]
I haven't started using them, but intend too with the new game. Will update when I figure it out!
